lms/djangoapps/certificates/views.py

```diff
@@ -591,6 +597,16 @@ def render_html_view(request, user_id, course_id):
     # Append/Override the existing view context values with any course-specific static values from Advanced Settings
     context.update(course.cert_html_view_overrides)
 
+    # track certificate evidence_visited event for analytics when certificate_user and accessing_user are different
+    if request.user and request.user.id != user.id:
+        emit_certificate_event('evidence_visited', user, course_id, course, {
```

Comment:
@mulby if i remove the explicit context then it would automatically populate the user who made the request from `request.user` so we don't need to change the argument here?.